apps/barista-design-system/src/pages/single-page/components/toc/toc.ts

```diff
@@ -40,15 +40,16 @@ import { DOCUMENT } from '@angular/common';
   },
 })
 export class BaToc implements OnInit, AfterViewInit, OnDestroy {
+  @Input()
+  tocitems: TOC;
```

Comment:
NIT: `tableOfContentItems` – as stated in our coding standards we don't use abbreviations.

Furthermore please add JsDoc